src/test/java/com/wikia/webdriver/common/clicktracking/ClickTrackingSupport.java

```diff
@@ -9,21 +9,17 @@
 
 public class ClickTrackingSupport {
 
-  public void compare(List<JsonObject> expectedEventList, List<JsonObject> currentEventList) {
+  public void compare(List<JsonObject> expectedEventList, List<JsonObject> trackedEventList) {
     for (JsonObject expectedEvent : expectedEventList) {
       Boolean equals = false;
-      for (JsonObject currentEvent : currentEventList) {
-        PageObjectLogging.log("compare",
-                              "comparing clicktracked events to expected event: \n"
-                              + "expected event: " + expectedEvent.toString() + "\n"
-                              + "compared event: " + currentEvent.toString(), true);
+      for (JsonObject currentEvent : trackedEventList) {
         equals = currentEvent.equals(expectedEvent);
         if (equals) {
           PageObjectLogging.log("compare",
```

Comment:
Remove PageObjectLogging, and log unexpected value in assertion below second if block